### ISSUE TITLE
EmptyComment reports empty lines in multi-line `//`

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#298]
Consider this comment from Doctrine:
```php

                // In some scenarios, such as EAGER+ASSOCIATION+ID+CACHE, the
                // cache key information in `$cacheEntry` will not contain details
                // for fields that are associations.
                //
                // This means that `$data` keys for some associations that may
                // actually not be cached will not be converted to actual association
                // data, yet they contain L2 cache AssociationCacheEntry objects.
                //
                // We need to unwrap those associations into proxy references,
                // since we don't have actual data for them except for identifiers.
                if ($unCachedAssociationData instanceof AssociationCacheEntry) {
```

Both those `//` are reported as empty comment. In my opinion, this should be valid use case and not be reported.

**Comment by kukulich** [comment]
Not sure. Should not be changed to block comment instead?

**Comment by Majkl578** [comment]
Since this type of comments is often also used to comment out PHP code, which typically can contain blank lines too, I would consider this multi-line comment and not report an error. (Unlike the case where empty line is not commented out.)
cc @ocramius @alcaeus, what do you think?

**Comment by alcaeus** [comment]
> what do you think

I’d consider a commented-out block of code including empty lines a checkstyle itself. I’m ok leaving it as is. // is for one line comments, for everything else, there are block comments

**Comment by Majkl578** [comment]
Maybe in the case above it would be better as multi-line `/* ... */` comment. But not in general.

What I don't like is this:
I have a valid code according to CS:
```
public function ($foo, $bar)
{
    parent::__construct($foo);

    $this->bar = $bar;
}
```
Now when I comment it out (default IDE behavior is commenting out with `//`, which makes it more obvious as well), it becomes invalid:
```php
//public function ($foo, $bar)
//{
//    parent::__construct($foo);
//
//    $this->bar = $bar;
//}
```

**Comment by alcaeus** [comment]
I understood your point without explanation. I just think commenting out code like that should be a temporary action. I wouldn't want to see a commit containing code like that, which is why I don't mind `phpcs` complaining about it. That's just my two cents here though.

**Comment by kukulich** [comment]
I agree with @alcaeus that commented code should be a temporary action but it will not cause any harm if the sniff will ignore these errors.

This should be ignored now:
```
// Comment
//
// Comment
```

This should be still reported:
```
// Comment
//

//
// Comment
```
